sktm/patchwork.py

```diff
@@ -330,7 +330,7 @@ def get_patch_message(self, patch_id):
 
         return email.message_from_string(response.content)
 
-    def get_header_values_all(self, patch_id, *name_tuple):
+    def __get_header_values_all(self, patch_id, *name_tuple):
```

Comment:
I wouldn't mark this method as private. Right now, it's not used outside of parent class methods, but both this and `get_header_values_first` are meant to be freely used in child classes.